python/cuml/preprocessing/LabelEncoder.py

```diff
@@ -183,7 +184,7 @@ def fit(self, y, _classes=None):
         self._fitted = True
         return self
 
-    def transform(self, y: cudf.Series) -> cudf.Series:
+    def transform(self, y) -> cudf.Series:
```

Comment:
```suggestion
    def transform(self, y):
```

Reply:
thank you so much. I'm making changes accordingly.